app/services/importers/ok_sos/base_importer.rb

```diff
@@ -9,7 +9,7 @@ class BaseImporter < ApplicationService
 
       def initialize(file_path)
         @file_path = file_path
-        @model_cache = ActiveSupport::HashWithIndifferentAccess.new({})
+        @@model_cache = ActiveSupport::HashWithIndifferentAccess.new({})
```

Comment:
We have to be a bit careful about this because once the cache is generated no changes will be reflect even between the different importer files